react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable prefer-const */
+import React, { useState } from 'react';
+
+export default function Accordion({ inputData }) {
+  const [activeIndex, setActiveIndex] = useState(undefined);
+
+  return (
+    <>
+      {inputData.map((entry) => Bellow(entry, activeIndex, setActiveIndex))}
+    </>
+  );
+}
+
+function Bellow({ title, id, details }, activeIndex, setActiveIndex) {
```

Comment:
Interesting. This is why you call it like you do up above. Technically, `Bellow` is not a React component because it takes 3 parameters. All React components must take exactly one parameter -- an object of properties.